content/doc/developer/views/symbols.adoc

```diff
@@ -0,0 +1,60 @@
+---
+title: Symbols
+layout: developerguide
+---
+
+image::/images/developer/views/symbols.svg[Selection of Symbols]

```

Comment:
May be a better rendering but this should be called out in the version it becomes available in
```suggestion
Symbols are available since Jenkins 2.TODO.

image::/images/developer/views/symbols.svg[Selection of Symbols]

```